packages/eslint-mdx/src/traverse.ts

```diff
@@ -56,28 +56,37 @@ export class Traverse {
           // prettier-ignore
           /* istanbul ignore next */
           else if (
-            !isComment(value) &&
-            !isSelfClosingTag(value) &&
-            !isOpenCloseTag(value)
+            isComment(value) ||
+            isSelfClosingTag(value) ||
+            isOpenCloseTag(value)
           ) {
+            jsxNodes.push(node)
+          } else {
+            // #272, we consider the first jsx node as open tag although it's not precise
+            if (!index) {
+              offset++
+            }
```

Comment:
I'm not quite following this, is this an needed because MDX 1 does not having estree nodes?

Reply:
Yes, `<div><p></p>` is parsed as a single jsx node, but I imagined it should be two jsx nodes `<div>` and `<p></p>`